dist/ledger/README-cli.md

```diff
@@ -361,3 +361,146 @@ This concludes the upgrade process. The device is now ready to be used with the
+   first thing to do is to perform a factory reset. To do this, follow the exact
+   sequence of button presses below:
+
+   - Press the Right button once.
+   - Press both buttons at the same time.
+   - Press both buttons at the same time again.
+   - Press the Right button once.
+   - Press both buttons at the same time.
+   - Press the Right button once.
+
+   After this sequence, the device will reboot and we have to access the Recovery
+   Mode again. Repeat steps 1 and 2.
+
+4. After the device is in Recovery Mode, it is ready for the onboarding process.
+   To start the onboarding, issue the following command in the terminal:
+
+   ```bash
+   /path/to/dist> ./setup-new-device
+   ```
+
+   You will see the output:
+
+   ```
+   Welcome to the Ledger Nano S powHSM Setup for RSK 
+
+   Connect your ledger into recovery mode:
+   Connect it while keeping the right button pressed until you see a Recovery message, then
+   release the right button and wait until the menu appears.
+   Press [Enter] to continue
+   ```
+
+5. Press `Enter` to proceed with the onboarding process. You will now notice that
+   the script will stop at this point waiting for user confirmation:
+
+   ```
+   Removing the Bitcoin App...
+   The Ledger will prompt for 'Allow Unknown Manager'. Please accept it.
+   If the Ledger prompts for 'Remove app' followed by the app name and identifier, then please accept it.
+   ```
+
+6. Press the Right button once to confirm allow access to the manager. The script
+   will continue with the onboarding process and will stop again at the following
+   step:
+
+   ```
+   Removing the Ethereum App...
+   If the Ledger prompts for 'Remove app' followed by the app name and identifier, then please accept it.
+   Removing the Fido App...
+   If the Ledger prompts for 'Remove app' followed by the app name and identifier, then please accept it.
+   Removing the existing certification authority (if any)...
+   If the Ledger prompts for 'Revoke certificate' followed by the certificate name and its public key, then please accept it.
+
+   Setting up the RSK certification authority...
+   The Ledger will prompt for 'Trust certificate' followed by the certificate name and its public key. Please accept it.
+   ```
+
+7. Press the Right button once to confirm that the certificate should be trusted.
+   The script will continue installing the apps and should only stop at the
+   following step:
+
+   ```
+   Installing the RSK Signer App...
+   Installing the RSK UI...
+
+   App installation complete. Please disconnect and reconnect the device.
+   You should see a white screen upon restart.
+   Press [Enter] to continue
+   ```
+
+8. After this point, all the information required to perform the onboarding is
+   provided by the script. The following steps are listed in the [Onboard the device](#onboard-the-device)
+   section of this document.
+
+### powHSM manager fails to start after finishing the onboarding process
```

Comment:
None of the items from this one onwards actually belongs here, mainly due to lack of context. I would just remove them. Maybe keep them somewhere to add on the appropriate repository.